src/algebra/lie/abelian.lean

```diff
@@ -199,12 +199,17 @@ def maximal_trivial_linear_map_equiv_lie_module_hom :
   ((maximal_trivial_linear_map_equiv_lie_module_hom f) : M → N) = f :=
 by { ext, refl, }
 
-@[simp] lemma coe_maximal_trivial_linear_map_equiv_lie_module_hom_apply
+@[simp] lemma coe_maximal_trivial_linear_map_equiv_lie_module_hom_symm
```

Comment:
Feel free to rename things to `max_triv_submodule` in a future PR (-;

Reply:
Will do, thanks for the review!